src/opdracht2.js

```diff
@@ -0,0 +1,127 @@
+import axios from 'axios';
+
+
+// sla de referentie naar het formulier op en plaats er een submit-event listener op
+// const searchForm = document.getElementById('search-form');
+// searchForm.addEventListener('submit, searchCountry');
+
+// 2. Schrijf een asynchrone functie die, met behulp van Axios, een GET-request maakt naar het juiste endpoint
+// voor `nederland`. Log de response in de console en bestudeer de data goed: hoe is het opgebouwd?
+const searchForm = document.getElementById('search-form');
+searchForm.addEventListener('submit', searchCountry);
+
+const countryInfoBox = document.getElementById('search-result');
+const errorMessageBox = document.getElementById('error-message');
+
+function searchCountry(err) {
+    err.preventDefault();
+    const queryfield = document.getElementById('query-field')
+
+    fetchCountryData(queryfield.value);
+    queryfield.value = '';
+}
+
+async function fetchCountryData(name) {
+
+    const previousSearchResult = document.getElementById('search-result-identifier');
+    const previousErrorMessage = document.getElementById('error-message-identifier');
+
+    if (previousErrorMessage) {
+        errorMessageBox.removeChild(previousErrorMessage);
+    }
+
+    if (previousSearchResult) {
+        countryInfoBox.removeChild(previousSearchResult);
+    }
+
+    const URI = `https://restcountries.com`
+    const ENDPOINT = `/v2/name/${name}`
```

Comment:
Variabelen in camelCase schrijven
```suggestion
    const uri = `https://restcountries.com`;
    const endpoint = `/v2/name/${name}`;
```